neo/tools/debugger/debugger.cpp

```diff
@@ -163,9 +167,16 @@ bool DebuggerServerInit ( void )
 		gDebuggerServer = NULL;
 		return false;
 	}
+	
+	com_enableDebuggerServer.ClearModified( );
```

Comment:
if we returned early because of some error or whatever, this cvar remains modified.
not sure this is desirable (would retry each frame until it succeeds, if it ever does..)

Reply:
i think we settled on this being the desirable behaviour.

Reply:
are you sure?
a common error case will be that `gDebuggerServer->Initialize()` fails because port `27980` is still blocked, for example when running two dhewm3 instances.
in that case gDebuggerServer will be newly allocated and then deleted again in every frame